pytest/lib/cluster.py

```diff
@@ -123,7 +124,18 @@ def get_validators(self):
         return self.json_rpc('validators', [None])
 
     def get_account(self, acc, finality='optimistic'):
-        return self.json_rpc('query', {"request_type": "view_account", "account_id": acc, "block_id": None, "finality": finality})
+        return self.json_rpc('query', {"request_type": "view_account", "account_id": acc, "finality": finality})
+
+    def get_access_key_list(self, acc, finality='optimistic'):
+        return self.json_rpc('query', {"request_type": "view_access_key_list", "account_id": acc, "finality": finality})
+
+    def get_nonce_for_pk(self, acc, pk, finality='optimistic'):
+        for access_key in self.get_access_key_list(acc, finality)['result']['keys']:
```

Comment:
this is unnecessary. You can query an access key directly with `"request_type": "view_access_key"`